pkg/frontend/mysql_cmd_executor.go

```diff
@@ -3626,6 +3626,11 @@ func (mce *MysqlCmdExecutor) doComQuery(requestCtx context.Context, input *UserI
 				}
 				canCache = false
 			}
+			if _, ok := cwft.stmt.(*tree.Select); ok {
```

Comment:
what if the statment is insert that current_timestamp is in values?

and strings.Contains -- what if it is upper case?   Finally if the current_timestamp is in a view definition, what will you do?    Are there other functions, for example, the function now?

this really should be walk on the tree or plan, or during plan generation, we should figure out some function is not cachable, whenever we run into that function we mrk it.

Reply:
ok